### PR TITLE
Ca addition

### PR DESCRIPTION
**Pull request by deltadstroyer**
Added the option to "/announce events" for a "Combined Arms" event
Added a bit of text to where it goes searching for the splash art so you can find it easier :P

### CODE REVIEW

File: src/commands/ops.py [hunk 1]
```diff
@@ -25,7 +25,8 @@ async def event_message(
         "FUEL": {"short_title":"FUEL", "ts_channel": "FUEL (Emerging Leaders)", "game": "Planetside 2", "ping_role": "Planetside 2", "color": 0x9E0B0F},
         "FUGG": {"short_title":"FUGG", "ts_channel": "FUGG (Galaxy Group)", "game": "Planetside 2", "ping_role": "Planetside 2", "color": 0x9E0B0F},
         "Huntsmen": {"short_title":"Huntsmen", "ts_channel": "Huntsmen", "game": "Planetside 2", "ping_role": "Planetside 2", "color": 0x9E0B0F},
-        "ArmaOps": {"short_title":"ArmaOps", "ts_channel": "", "game": "Arma 3", "ping_role": "Arma 3", "color": 0xb641d}
+        "ArmaOps": {"short_title":"ArmaOps", "ts_channel": "", "game": "Arma 3", "ping_role": "Arma 3", "color": 0xb641d},
+        "CombinedArms": {"short_title":"CombinedArms", "ts_channel": "Combined Arms", "game": "Planetside 2", "ping_role": "Planetside 2", "color": 0x9E0B0F}
```
Comment (karlis-ozo) [inline src/commands/ops.py:29]: I believe this should be ` "ts_channel": "Outfit Drill"` as Combined Arms doesn't have a TS channel

**Review by karlis-ozo — CHANGES_REQUESTED** [review]
Looks good. but there isn't such a TS channel at this time. we should ether use Drill channel or create a new channel.

also perhaps ask Delfi for new images? he will need the sutext as well though. But shouldn't be that bad if we just reuse the drill ones

File: src/commands/ops.py (hunk 1 above)
Comment (deltadstroyer) [reply src/commands/ops.py:29]: Only issue is that the first line of text looks at the "ts_channel" for input, so using "Outfit Drill" makes the first line of the announcement "Outfit Drill - Starting NOW!"
Making a TS channel should not be that hard....

**Comment by karlis-ozo** [comment]
This has been added to #100 and merged in develop

**Pull request closed without merging**